tilejson/tilejson.json.erb

```diff
@@ -17,7 +17,7 @@
     "name" : "<%= @params[:name] %>",
     "private" : true,
     "scheme" : "xyz",
-    "tilejson" : "2.0.0",
+    "tilejson" : "<%= @params[:tilejson_version] %>",
     "tiles" : [
         "http://<%= node[:tilejson][:host] %>/mapzen/vector/<%= node[:tilejson][:vector_tiles_version] %>/all/{z}/{x}/{y}.<%= @params[:extension] %>?api_key={api_key}"
```

Comment:
Nice catch, updated in b18f1b9.

I also noticed that I was referencing the api version incorrectly too. Should be fixed in 82aa063.